src/property-filter/__integ__/async-loading.test.ts

```diff
@@ -206,3 +209,24 @@ test.each<TestCase>(testCases)('%p', (_, asyncProperties, token, scenario) =>
     }
   })()
 );
+
+test(
+  'calls onLoadItems with an empty string if Clear button is clicked',
+  setupTest(true, 'property', async page => {
+    await page.typeInFilteringInput('1');
+    await page.expectLoadItemsEvents([
+      { firstPage: true, samePage: false, filteringText: '' },
+      { filteringText: '1', firstPage: true, samePage: false },
+    ]);
+
+    await page.click('#focus-target');
+    await page.clearFilteringInput();
+    await page.waitForJsTimers(500);
+    await page.expectLoadItemsEvents([
+      { firstPage: true, samePage: false, filteringText: '' },
+      { filteringText: '1', firstPage: true, samePage: false },
+      { filteringText: '1', firstPage: true, samePage: false },
```

Comment:
why is this event repeating?

Reply:
first two calls for the first `onLoadItems`, consequent for the load after clicking on clear. So, when you click on clear, `onLoadItems` is called twice, because of `focus` event first, and afterwards after clearing the input

Reply:
I did some testing and here is what I found:

When using same test configuration and selecting `label =` as input and then clearing it - the following events are produced:
```
{filteringText: '', samePage: false, firstPage: true} // focus
{filteringText: 'label', samePage: false, firstPage: true} // enter label
{filteringText: '', samePage: false, firstPage: true, filteringProperty: {…}, filteringOperator: '='} // enter operator
{filteringText: '', samePage: false, firstPage: false} // clear 1
{filteringText: '', samePage: false, firstPage: true} // clear 2
```

I think only one is expected (that is what happens with clearing with keyboard).

When using test configuration with asyncProperties=false, there is no even when the search is cleared.

Reply:
Also, let's add the test for clearing the input with the keyboard (by selecting and clearing the text). In the current mainline there is no event triggered in that case but that is not supported with the current fix.

Reply:
Idea: can we resolve the issue by sending no event when the clear button is clicked? When the search input is empty the event is not really needed as there is nothing to load.